hcl/gapi_test.go

```diff
@@ -0,0 +1,7 @@
+package hcl
+
+import "testing"
+
+func TestGAPI(t *testing.T) {
+
```

Comment:
Was this intentionally left as a stub, or did you forget to commit something in it?